tests/target/issue-2321/trailing_comma_always_and_brace_style_same_line_where.rs

```diff
@@ -0,0 +1,10 @@
+// rustfmt-brace_style: SameLineWhere
+// rustfmt-trailing_comma: Always
+
+fn lorem<S, T,>(lorem: S, ipsum: T,)
+where
+    S: Add + Sub,
+    T: Mul + Div,
+{
```

Comment:
Maybe I don't understand the `brace_style` config fully, but with an option like `SameLineWhere` I'd expect this to format with the opening brace on the same line as the last where clause bound. Do you think the current behavior is correct?